dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -602,6 +593,19 @@ private void setWorkflowInfo(final String repositoryId, final String type, final
             logAndThrowLambdaFailure(String.format("Type %s is not a valid descriptor language.", type));
         }
 
+        workflow.setWorkflowName(workflowName);
+        workflow.setOrganization(repositoryId.split("/")[0]);
+        workflow.setRepository(repositoryId.split("/")[1]);
+        workflow.setSourceControl(SourceControl.GITHUB);
+        workflow.setGitUrl("git@github.com:" + repositoryId + ".git");
+        workflow.setLastUpdated(new Date());
+        workflow.setDefaultWorkflowPath(DOCKSTORE_YML_PATH);
```

Comment:
I debated whether to instead change the `setDefaultWorkPath` implementation (there's a TODO on the method) instead of reordering lines, so it wouldn't be dependent on `setDescriptorType()` having been invoked first, but it seemed riskier, especially if we wanted to hotfix this (unlikely, but who knows).